marshmallow/fields.py

```diff
@@ -1172,13 +1173,13 @@ def _deserialize(self, value, attr, data, **kwargs):
             self.fail('invalid')
 
 
-class Dict(Field):
-    """A dict field. Supports dicts and dict-like objects. Optionally composed
+class Mapping(Field):
+    """An abstract class for objects with key-value pairs. Optionally composed
     with another `Field` class or instance.
```

Comment:
> Optionally composed with another `Field` class or instance.

I realize you didn't add this, but it should be corrected, since `Mapping` classes can be composed with both a "key" and a "value" container. How about we just drop that sentence?

Reply:
Done